src/components/Loader/Loader.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import { Meta } from '@storybook/react';
+import Loader from './Loader';
+
+const meta = {
+  title: 'Components/Loader',
+  component: Loader,
+} satisfies Meta<typeof Loader>;
```

Comment:
Can you add `argTypes` to this so that args can be set in the story? It should be something like this (this is from memory, the syntax may not be 100% correct):

```
const meta = {
  title: 'Components/Loader',
  component: Loader,
  argTypes: { 
    isCentered: { 
      control: 'boolean'
    }
  }
} satisfies Meta<typeof Loader>;